Wesley so wins final leg of fide grand prix series (#1681)
slurpcode/slurp: doc/source/wesley-so-wins-final-leg-of-fide-grand-prix-series.md

> # ♟️ Wesley So wins final leg of FIDE grand prix series, Hikaru and Richard qualifies for Candidates

Reviewer: ```suggestion
# ♟️ Wesley So wins final leg of FIDE grand prix series, Hikaru and Richard qualify for Candidates
```

and half : half and won the third leg of the Grand Prix in Berlin. The
tournament winner and the runner-up looked happy at the closing ceremony
as both of them were successful in the major tournament and world
championship cycle. Despite the loss in the final match, Hikaru Nakamura
qualified for the FIDE Candidates Tournament 2022 and became the overall
winner of FIDE Grand Prix Series 2022. Richard Rapport, who is the
second in the overall standings and also qualified for the Candidates,
while Wesley So was third in overall Series.<br><br>

After the match ended, Wesley So said: \"I am only 28, and I\'m hoping
that next year or in a couple of years, I will get a chance to play in
the Candidates. The last time I played, I was very inexperienced and
finished second to last, so I think if you qualify, you have to be ready
to fight for first place.\"
<br><br>
Hikaru Nakamura: \"The odds [of winning the whole series] were
probably 10-15% before everything began. I was quite fortunate in the
first leg as I got players who hadn\'t played as much; they weren\'t
sharp.\"
<br><br>
In the first tiebreaker game, Hikaru played the Bishop opening with

> white, which was a shocker to Wesley So as he didn\'t expect this to see

Reviewer: ```suggestion
white, which was a shocker to Wesley So as he didn\'t expect to see this
```

on the board today. However, Wesley played confidently with Black and

> got a some decisive positional advantage by move 15. Hikaru showed his

Reviewer: ```suggestion
got a decisive positional advantage by move 15. Hikaru showed his
```

excellent defensive skills and tenaciousness, and after a couple of

> mistakes by the wesley, he managed to create a very dangerous attack on

Reviewer: ```suggestion
mistakes by Wesley, he managed to create a very dangerous attack on
```

|   Maxime Vachier-Lagrave   |        |   7    |   1    |   8       |
|   Anish Giri               |        |   7    |   0    |   7       |
|   Vidit Gujrathi           |   3    |        |   4    |   7       |
|   Nikita Vitiugov          |        |   3    |   3    |   6       |
|   Alexandr Predke          |        |   3    |   1    |   4       |
|   Grigoriy Oparin          |   0    |        |   4    |   4       |
|   Andrey Esipenko          |   4    |        |   0    |   4       |
|   Vincent Keymer           |   0    |        |   4    |   4       |
|   Radosław Wojtaszek       |   4    |        |        |   4       |
|   Vladimir Fedoseev        |   2    |   1    |        |   3       |
|   Yu Yangyi                |        |   0    |   3    |   3       |
|   Daniil Dubov             |   3    |        |   0    |   3       |
|   Alexander Grischuk       |   2    |   0    |        |   2       |
|   Pentala Harikrishna      |   2    |   0    |        |   2       |
|   Etienne Bacrot           |   0    |   2    |        |   2       |
|   Alexei Shirov            |   0    |   1    |        |   1       |

## About the Author

> Ashwaneel Atil is a student pursuing Bachelor of Engineering and a young

Reviewer: ```suggestion
Jashwaneel Patil is a student pursuing Bachelor of Engineering and a young
```
